apps/studio/prisma/schema.prisma

```diff
@@ -30,6 +30,7 @@ generator kysely {
 model VerificationToken {
   identifier String   @id
   token      String
+  prefix     String
```

Comment:
are these the alphabets in front of the OTP numbers?

Reply:
Clarifying here, are you suggesting that we store the unhashed OTP or the hashed  version? I didn't want to store the unhashed version, hence I stored the prefix as an identifier

Reply:
i'm saying maybe accept the DOS risk and just keep old behavior. Just add rate limiting to mitigate DOS ability

Reply:
i'm ok with this. in this case, i'll revert the changes in my last commit. under this model, it's possible for a malicious actor to prevent a good user from logging in purely by persistently re-requesting otp. if we're ok with this trade-off, i'll make the changes